### PR TITLE
Delaying of PATH_RESPONSE

### PR DESCRIPTION
**Pull request by mirjak**
This change was discussed as part of PR #388

### CODE REVIEW

**Review by huitema — COMMENTED** [review]
The text is a bit hard to follow until we check in the other PR on denying path!

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -387,19 +387,18 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
+An endpoint that lacks resource to immediately configure a new path MAY
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:390]:
```suggestion
An endpoint that lacks resource to immediately configure a new path could
```

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -387,19 +387,18 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
+An endpoint that lacks resource to immediately configure a new path could
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:390]:
```suggestion
An endpoint that has no active connection ID for this path or
lacks other resource to immediately configure a new path could
```
